lib/platform/github/gh-graphql-wrapper.ts

```diff
@@ -71,13 +75,13 @@ export class graphql {
 
       if (!res.data) {
         // retry query until numElements == 1
-        if (repoResultNumEls === 1) {
+        if (this.repoResultNumEls === 1) {
           logger.info({ query, res }, 'No graphql res.data');
           return [false, [], ''];
         }
 
         this.repoResultNumEls = parseInt(this.repoResultNumEls / 2);
-        return this.get(cursor);
+        return await this.get(cursor);
```

Comment:
You shouldn't normally need to `return await` - it's considered an antipattern. Is this to make it easier to test?

Reply:
Uhmm, it just made sense that it should be like that. Doesn't it return the `Promise` object if no `await` keyword is passed? Got any reference regarding this so I can check out?